api/envoy/config/route/v3/route_components.proto

```diff
@@ -2168,16 +2169,47 @@ message RateLimit {
     }
   }
 
+  message HitsAddend {
```

Comment:
should this look more like https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/core/v3/base.proto#config-core-v3-datasource ?

Reply:
sorry, but I didn't get what you mean. 🤣 This HitsAddend is only used to configure single fixed number of a dynamic formatter provider.

Reply:
yeah what I was getting at, is there any anything existing thats reusable ?

Reply:
for e.g. the style used here https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/http/custom_response/local_response_policy/v3/local_response_policy.proto#extensions-http-custom-response-local-response-policy-v3-localresponsepolicy is `body` and `body_format`

Reply:
> yeah what I was getting at, is there any anything existing thats reusable ?

Nope for now. And I think this is a little special may not deserve to make it reusable.